src/NServiceBus.Core.Tests/Msmq/MsmqUtilitiesTests.cs

```diff
@@ -18,7 +21,7 @@ public void Should_convert_a_message_back_even_if_special_characters_are_contain
         {
             var expected = String.Format("Can u see this '{0}' character!", (char)0x19);
             
-            var options = new TransportSendOptions("destination");
+            var options = new DispatchOptions("destination", new AtomicWithReceiveOperation(), new List<DeliveryConstraint>());
```

Comment:
Not sure that it would add much to this test. What I've done instead is to make the default consistency guarantee explicit (transports have to provide one)

https://github.com/Particular/NServiceBus/commit/73b253f06c77c3128e2d89b9c5c134af742a0ffd

Does this make sense?